### PR TITLE
Add stdout/stderr support in CommandEventContext

### PR DESCRIPTION
**Pull request by filipgolem**

### CODE REVIEW

File: yajsapi/executor/index.ts [hunk 1]
```diff
@@ -598,7 +598,7 @@ export class Executor {
             emit(evt);
             results.push(evt);
             if (evt instanceof events.CommandExecuted && !evt.success) {
-              throw new CommandExecutionError(evt.command, evt.message)
+              throw new CommandExecutionError(evt.command, evt.stderr)
```
Comment (azawlocki) [inline yajsapi/executor/index.ts:601]: In `yapapi`, `CommandExecutionError` now has both `message` and `stderr` fields. You may want to change it also in the JS version.

Comment (azawlocki) [reply yajsapi/executor/index.ts:601]: If not, I'd suggest leaving `evt.message` as the second argument of the constructor.

**Review by azawlocki — CHANGES_REQUESTED** [review]

File: yajsapi/executor/index.ts (hunk 1 above)
Comment (filipgolem) [reply yajsapi/executor/index.ts]: Fixed

**Review by azawlocki — APPROVED** [review]
LGTM!

**Pull request merged**